### DISCUSSION TITLE
Can't find zest API Document

### DISCUSSION THREAD

**Discussion by indie2k** [eclipse-gef/gef-classic#145] [Q&A]
Hello.

I'm looking for Zest API Documents ( GEF-Classic)

I could find GEF(MVC) API Documents but not Zest API even in 'org.eclipse.zest.doc.isv' project's contents

Anyone can help me? 

(+ I'm not good at English ^^)

**Comment by vogella** [comment]
Not sure if we have any API documentation for Zest. @azoitl do you know?

**Comment by azoitl** [comment]
As I understand it the API documentation should be generated during the build from Java doc. Isn't it then part of the distribution?

**Reply by indie2k** [reply]
the API documentation should be generated during the build from Java doc <- yes. I think so. 
But seems like not part of the distribution now.

in Eclipse Documents (https://help.eclipse.org/2023-03/index.jsp)
When go to - GEF Classic Developer Guide -> Reference -> API Reference -> Zest, there is no api documents,
but GEF(MVC) exists.

but I found there had been Zest documents until 2022-06 Document ( https://help.eclipse.org/2022-06/index.jsp )
GEF Developer Guide -> Reference -> API Reference -> Zest, There is the API Documents.
